.github/workflows/main.yml

```diff
@@ -36,4 +36,4 @@ jobs:
         version: latest
         args: release --rm-dist
       env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        GITHUB_TOKEN: ${{ secrets.JASONBLANCHARD_TMP_REPO_RW }}
```

Comment:
Making it painfully clear that we're using a personal access token tied to my account right now. I think we just need to make a machine account for things like this and use personal access tokens tied to that.